deps/rabbit/src/rabbit_disk_monitor.erl

```diff
@@ -264,11 +250,46 @@ parse_free_unix(Str) ->
         _          -> exit({unparseable, Str})
     end.
 
-parse_free_win32(CommandResult) ->
-    LastLine = lists:last(string:tokens(CommandResult, "\r\n")),
-    {match, [Free]} = re:run(lists:reverse(LastLine), "(\\d+)",
-                             [{capture, all_but_first, list}]),
-    list_to_integer(lists:reverse(Free)).
+win32_get_disk_free_fsutil(Dir) ->
+    % Dir:
+    % "c:/Users/username/AppData/Roaming/RabbitMQ/db/rabbit2@username-z01-mnesia"
+    Drive = string:slice(Dir, 0, 2),
```

Comment:
I don't think this is going to work for network volumes as those do not have a letter. Just a heads up, I do not know if that was working before. If they are not desirable I would do a `Dir = [_, $:, $/|_]` in the function head. Same for the other command.

Reply:
It would not have worked before, correct. I will add the check.